src/relay-runtime/index.d.ts

```diff
@@ -0,0 +1,3 @@
+// @flow
```

Comment:
`@flow` in `.ts` file? 🤔

Reply:
Hah, that's an interesting glitch in our Eslint config, see #320